cirq-superstaq/cirq_superstaq/daily_integration_test.py

```diff
@@ -286,13 +287,24 @@ def test_submit_to_provider_simulators(target: str, service: css.Service) -> Non
     assert job.counts() == {"11": 1}
 
 
-def test_submit_to_hilbert(service: css.Service) -> None:
-    q0 = cirq.LineQubit(0)
-    q1 = cirq.LineQubit(1)
-    circuit = cirq.Circuit(cirq.X(q0), cirq.CNOT(q0, q1), cirq.measure(q0, q1))
+def test_submit_to_hilbert_qubit_sorting(service: css.Service) -> None:
+    """Regression test for https://github.com/Infleqtion/client-superstaq/issues/776"""
+    target = "cq_hilbert_qpu"
+    num_qubits = service.target_info(target)["num_qubits"]
+    qubits = cirq.LineQubit.range(num_qubits)
+    circuit = cirq.Circuit(
+        css.ParallelRGate(np.pi / 2, 0.0, 24).on(*qubits),
+        cirq.rz(np.pi).on(qubits[2]),
+        css.ParallelRGate(-np.pi / 2, 0.0, 24).on(*qubits),
+        cirq.measure(*qubits),
+    )
 
-    job = service.create_job(circuit=circuit, repetitions=100, target="cq_hilbert_qpu")
-    assert sum(job.counts().values()) == 100
+    job = service.create_job(
+        circuit=circuit, repetitions=400, verbatim=True, route=False, target=target
```

Comment:
do we need this many shots? it seems like 100 should still be enough for this to pass consistently (though also not sure if there's any reason to use fewer)

Reply:
Changed to `100`